README.md

```diff
@@ -1 +1,4 @@
-# review_trial
\ No newline at end of file
+# review_trial
```

Comment:
- Write a more descriptive readme. Tell us what your project is about, tools used, how to get started, running locally. These sections make your repo look professional and easy to get started with. 
Checkout this [template](https://github.com/microverseinc/readme-template) for reference.

Reply:
- [ ] I think it will sound friendlier, softer and professional to rephrases as: Please add a more descriptive readme file as this will enable other users, potential teammates and recruiters to know what the project is about, and how to run it. The readme file will also communicate to potential recruiters the type of person  you are. It is always good to avoid imperative tenses when giving feedback.